tests/everflow/everflow_test_utilities.py

```diff
@@ -273,7 +361,7 @@ def add_route(duthost, prefix, nexthop, namespace):
         namespace: namsespace/asic to add the route
 
     """
-    duthost.shell(duthost.get_vtysh_cmd_for_namespace("vtysh -c \"configure terminal\" -c \"ip route {} {}\"".format(prefix, nexthop), namespace))
+    duthost.shell(duthost.get_vtysh_cmd_for_namespace("vtysh -c \"configure terminal\" -c \"ip route {} {} tag 1\"".format(prefix, nexthop), namespace))
```

Comment:
Why do we need `tag 1` here?

Reply:
@bingwang-ms  : Check this: https://github.com/sonic-net/sonic-mgmt/pull/4721